conanfile.py

```diff
@@ -55,7 +56,13 @@ class VehicleAppCppSdkConan(ConanFile):
     def set_version(self):
         try:
             git = tools.Git(folder=".")
-            version = git.get_tag() if git.get_tag() is not None else git.get_branch()
+            tag = git.get_tag()
+            if tag is not None:
+                version_tag_pattern = re.compile(r"^v[0-9]+(\.[0-9]+){0,2}$")
```

Comment:
I would leave off the final "$": Without we keep our option to have version tags like "v1.0.0-rc1" (or similar)
```suggestion
                version_tag_pattern = re.compile(r"^v[0-9]+(\.[0-9]+){0,2}")
```

Reply:
Done